sdk/search/azure-search-documents/azure/search/documents/_service/_models.py

```diff
@@ -0,0 +1,105 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from ._generated.models import Analyzer, Tokenizer
+
+class PatternAnalyzer(Analyzer):
+    """Flexibly separates text into terms via a regular expression pattern.
```

Comment:
Nit: "expression pattern" is redundant. Just "expression" should be fine and is typical, e.g. "regular expression".

Reply:
This is from swagger too. :)